frontend/src/interfaces/uiConfig.ts

```diff
@@ -79,6 +79,7 @@ export type UiFlags = {
     executiveDashboard?: boolean;
     changeRequestConflictHandling?: boolean;
     feedbackComments?: Variant;
+    displayUpgradeEdgeBanner?: boolean;
```

Comment:
Might be worth considering making this a `Variant` flag instead. That way we can control not only whether the banner is shown, but also its contents through the flag.

Reply:
This is something we'd like to have in OSS deploys as well, and hopefully not for very long.

Reply:
So, I think having it as a normal flag is fine.